systemtests/src/test/java/io/enmasse/systemtest/isolated/api/CustomResourceDefinitionAddressSpacesTest.java

```diff
@@ -181,7 +179,6 @@ void testCreateAddressSpaceViaCmdNonAdminUser() throws Exception {
     }
 
     @Test
-    @DisabledIfEnvironmentVariable(named = OCP_VERSION_ENV, matches = "3.10")
```

Comment:
why you removed this?

Reply:
don't you think of adding @AssumeOpenshiftVersion(version = 3) ?

Reply:
because this test failed only on ocp 3.10 which is not supported anyway, so it was old disable annotation, this test works on minikube and ocp 3,4